### PR TITLE
Explain our position towards steem, ads, and other incentives

### PR DESCRIPTION
**Pull request by rigelk**
Ads, blockchain and creation incentives in general are recurring topics:
- https://framacolibri.org/t/blockchain-et-peertube/2587
- https://framacolibri.org/t/fini-framatube-peertube-posez-nous-vos-questions/1965/88
- https://framacolibri.org/t/fini-framatube-peertube-posez-nous-vos-questions/1965/100
- https://diaspodon.fr/@dada/99644524496798848
- (probably others I have forgotten)

Trying to make a coherent synthesis of the current state of our opinion is not an easy task, and I don't expect this PR to be perfect. In no way that means we have to stop talking about these subjets or close doors, but it certainly gives us material to start more interesting discussions in the future.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -50,3 +68,23 @@ So you would need:
...
+
+Long answer is that the Steem blockchain goes astray of its promises of fairness and decentralization: the deliberate relaunching of the currency to ensure centralization, and the stake-based voting power, makes manipulation by wealthy users inevitable ([source here](https://decentralize.today/the-ugly-truth-behind-steemit-1a525f5e156)). Worse, money generated goes primarly to stake holders ([source here](https://steemit.com/steemit/@orly/how-the-steem-pyramid-scheme-really-works) ). For more information, read the complete whitepaper analysis done by [Tone Vays](https://twitter.com/ToneVays/status/761975587451928576).
+
+## Are you gonna support adding ads?
+
+No. Short answer is that we don't want advertisers to dictate which content should be financed, thus modifying the dynamic of creation ; as such it cannot be integrated into mainline PeerTube. Read more about that in [the dedicated section](#what-is-creation-dynamic-and-why-not-modify-it).
```
Comment (Chocobozzz) [inline FAQ.md]: Maybe prefixing the line with **Short answer:** for better readability?

File: FAQ.md [hunk 2]
```diff
@@ -50,3 +68,23 @@ So you would need:
...
+
+The long answer is probably more subtle. YouTube has shaped generations of video creators by making it easy to place ads ; but making big money with the platform can be a challenge. A typical video ad runs between $.10 and $.30 per 1000 view (as of march 2018). More than 70% of video creators have ads as their main way to make money on YouTube, yet less than 3% of video creators make a living out of their YouTube activity (with partnerships and commissions, otherwise counting only ad revenue it drops to 1%). Read more about it in the 2018 study by Mathias Bärtl, [*YouTube channels, uploads and views: A statistical analysis of the past 10 years*](https://www.dropbox.com/s/0cq4wtxm83s95t2/10.1177%401354856517736979.pdf?dl=0). To the best of our knowledge, small and medium-community creators are better off getting support from their community on platforms such as Liberapay, Tipee or Patreon. Moreover, don't forget that advertisers already pay considering YouTube's large user base ; with PeerTube's way smaller user base and refusal of user profiling, a lower pay-per-view than on YouTube could only be expected.
```
Comment (ldidry) [inline FAQ.md]: That's "Tipeee", with 3 "e"

Comment (Chocobozzz) [reply FAQ.md]: Like Chocobozzz, with 3 "z"

Comment (rigelk) [reply FAQ.md]: Thanks for spotting the typo!

File: FAQ.md [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
+
+- [If nobody watches a video, is it seeded?](#if-nobody-watches-a-video-is-it-seeded)
+- [What is WebSeed?](#what-is-webseed)
+- [If a client requests each chunk of a video through HTTP, will the server be overloaded?](#if-a-client-requests-each-chunk-of-a-video-through-http-will-the-server-be-overloaded)
+- [Will an index of all the videos of servers you follow be too large for small servers?](#will-an-index-of-all-the-videos-of-servers-you-follow-be-too-large-for-small-servers)
+- [What codecs can I use for the videos I want to upload?](#what-codecs-can-i-use-for-the-videos-i-want-to-upload)
+- [I want to change my host, how can I do that?](#i-want-to-change-my-host-how-can-i-do-that)
+- [Should I have a big server to run PeerTube?](#should-i-have-a-big-server-to-run-peertube)
+- [Are you gonna use the Steem blockchain?](#are-you-gonna-use-the-steem-blockchain)
+- [Are you gonna support adding ads?](#are-you-gonna-support-adding-ads)
```
Comment (bnjbvr) [inline FAQ.md]: nit: `gonna` is familiar, please use `going to` instead (as well in anchors)

File: FAQ.md [hunk 4]
```diff
@@ -44,9 +62,29 @@ If you don't have any followers, update your configuration and run
...
  * **RAM** -> nginx ~ 6MB, peertube ~ 120MB, postgres ~ 10MB, redis ~ 5MB
- 
+
 So you would need:
  * **CPU** 1 core if you don't enable transcoding, 2 at least if you enable it
  * **RAM** 1GB
  * **Storage** Completely depends on how many videos your users will upload
 
+
+## Are you gonna use the Steem blockchain?
```
Comment (bnjbvr) [inline FAQ.md]: nit: `going to`

File: FAQ.md [hunk 5]
```diff
@@ -44,9 +62,29 @@ If you don't have any followers, update your configuration and run
...
+
+Short answer: no, since like most appchains/votechains, it modifies the dynamic of creation, and as such cannot be integrated into mainline PeerTube. Read more about that in [the dedicated section](#what-is-creation-dynamic-and-why-not-modify-it).
+
+Long answer is that the Steem blockchain goes astray of its promises of fairness and decentralization: the deliberate relaunching of the currency to ensure centralization, and the stake-based voting power, makes manipulation by wealthy users inevitable ([source here](https://decentralize.today/the-ugly-truth-behind-steemit-1a525f5e156)). Worse, money generated goes primarily to stake holders ([source here](https://steemit.com/steemit/@orly/how-the-steem-pyramid-scheme-really-works) ). For more information, read the complete whitepaper analysis done by [Tone Vays](https://twitter.com/ToneVays/status/761975587451928576).
```
Comment (bnjbvr) [inline FAQ.md]: nit: `primarily goes` (invert adverb and verb)

Comment (bnjbvr) [reply FAQ.md]: nit: `stakeholders` in one word

File: FAQ.md [hunk 6]
```diff
@@ -44,9 +62,29 @@ If you don't have any followers, update your configuration and run
...
+
+## Are you gonna support adding ads?
```
Comment (bnjbvr) [inline FAQ.md]: nit: `going to`

File: FAQ.md (hunk 3 above)
Comment (bnjbvr) [reply FAQ.md]: Also I'd rephrase the later to `are you going to support advertisements?`

File: FAQ.md [hunk 7]
```diff
@@ -44,9 +62,29 @@ If you don't have any followers, update your configuration and run
...
+
+Short answer: no, we don't want advertisers to dictate which content should be financed. That would modify the dynamic of creation ; as such it cannot be integrated into mainline PeerTube. Read more about that in [the dedicated section](#what-is-creation-dynamic-and-why-not-modify-it).
```
Comment (bnjbvr) [inline FAQ.md]: typo nit: no space before `;`.

File: FAQ.md [hunk 8]
```diff
@@ -44,9 +62,29 @@ If you don't have any followers, update your configuration and run
...
+
+The long answer is probably more subtle. YouTube has shaped generations of video creators by making it easy to place ads ; but making big money with the platform can be a challenge. A typical video ad runs between $.10 and $.30 per 1000 view (as of march 2018). More than 70% of video creators have ads as their main way to make money on YouTube, yet less than 3% of video creators make a living out of their YouTube activity (with partnerships and commissions, otherwise counting only ad revenue it drops to 1%). Read more about it in the 2018 study by Mathias Bärtl, [*YouTube channels, uploads and views: A statistical analysis of the past 10 years*](https://www.dropbox.com/s/0cq4wtxm83s95t2/10.1177%401354856517736979.pdf?dl=0). To the best of our knowledge, small and medium-community creators are better off getting support from their community on platforms such as Liberapay, Tipeee or Patreon. Moreover, don't forget that advertisers already pay considering YouTube's large user base ; with PeerTube's way smaller user base and refusal of user profiling, a lower pay-per-view than on YouTube could only be expected.
```
Comment (bnjbvr) [inline FAQ.md]: same typo nit again a few times here and below (no space before semicolon).

nit: `per 1000 views`
nit: `as of March 2018`
suggestion: `have ads as their main way` => `use ads as the main way`
suggestion: `a lower pay-per-view than on YouTube` => `a pay-per-view that's lower than YouTube's`

File: FAQ.md [hunk 9]
```diff
@@ -44,9 +62,29 @@ If you don't have any followers, update your configuration and run
...
+
+## What is "creation dynamic" and why not modify it?
+
+We define creation dynamic as the way any original content, regardless of its monetary value, is done and incentivized. We want to limit as much as possible the influence our platform could have on authors in order to stay neutral. We are not curators, and want to limit the scope of PeerTube's impact on videos to moderation tasks only.
```
Comment (bnjbvr) [inline FAQ.md]: suggestion: replace `done` by `created` in the first sentence?

suggestion: replace `want to limit the scope of PeerTube's impact on videos to moderation tasks only.` by `want to limit the scope of PeerTube owners' responsibilities to moderation tasks only`

Comment (rigelk) [reply FAQ.md]: Interestingly, I think your sentence suggestion better completes the preceding sentence of the paragraph, which focuses on the platform without considering the people managing it themselves. I would change "owner" with "instance owners and administrators", tough.

File: FAQ.md (hunk 3 above)
Comment (rigelk) [reply FAQ.md]: I wasn't sure about the "support advertisments" but now that I think of it, that's how everyone phrases it.

File: FAQ.md (hunk 5 above)
Comment (rigelk) [reply FAQ.md]: ah indeed! thanks :)

File: FAQ.md (hunk 7 above)
Comment (rigelk) [reply FAQ.md]: didn't know about the semicolons!

File: FAQ.md (hunk 8 above)
Comment (rigelk) [reply FAQ.md]: I corrected that, thanks! Your suggestions are less convoluted than mine, I'll use them.

**Comment by rigelk** [comment]
@bnjbvr I ended up doing sentence wraping, as we are mainly dealing with sentences as the smallest consistent unit of reasoning.

**Comment by mikelgs1** [comment]
I was just wondering that it'd be great if our public TV had a peertube instance, but I couldn´t even start promoting it if they can´t insert and manage adds. (if they are Public why do they need adds? Yeah, I know...). 

I am not thinking about adds focused on paying content creators (and controlling it doubtless, that is certain). I was suggesting that the instance owner gets money by inserting adds at the beginning of the videos (a public TV posting their own videos, the only users would be their TV shows). 

I think it's interesting to think about it, but probably not a priority at all, so... bye! 😁😘👋
